src/agent/coverage/tests/snapshot.rs

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
(The special `tests` crate is kinda sorta its own crate, from the perspective of export visibility, but not from the build and test-running perspective)

Reply:
Alternate idea: add a build feature that lets users set an env var to _opt out_ of these tests when desiring a quicker inner dev loop. Then this will still run by default without any extra work on the CI side.

Reply:
Some frameworks have a way to mark slow tests which don't run by default, but this doesn't seem to be available in Rust/nextest.

Reply:
Ah, sorry, I was thinking of something along the lines of [this existing test](https://github.com/microsoft/onefuzz/blob/d464e34a6fc03cc7fc561f07cab5750db3c9bde0/src/agent/onefuzz-task/src/tasks/fuzz/supervisor.rs#L320), which is gated on 
a non-default crate feature. Looks like you still can't yet do directly via an env var, but same idea.

Reply:
Actually, there is `#[ignore]` and nextest has `--run-ignored all` which we can turn on for the CI build. I'll just make sure that this doesn't break things through other ignored tests which are never meant to be run 🙂

Reply:
After some discussion we decided we wanted `#[ignore]` to actually ignore, so I've added a `slow-tests` feature instead, which is enabled during the CI build.